build/codegen/codegen.sh

```diff
@@ -14,29 +14,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-GROUP_VERSIONS="ceph.rook.io:v1"
+set -o pipefail
+set -o nounset
+set -o errexit
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
-scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+base_dir="$(cd "${script_dir}/../.."  && pwd)"
```

Comment:
Finding the repo base dir is IMO related to finding the script dir, so keep `script_dir` and `base_dir` lines together.

Reply:
applied